glossary.md

```diff
@@ -4,6 +4,7 @@ AKA the thing that no one ever reads. Below we've provided tweet-sized definitio
 
 - [**Capitalism**](https://en.wikipedia.org/wiki/Capitalism) an economic system based on private ownership of the means of production with the goal of generating profits. See: [The Black Socialists of America Glossary of Socialist Terms](https://blacksocialists.us/mumbo-jumbo)
 - [**Co-operative**](https://www.ica.coop/en/cooperatives/what-is-a-cooperative): a people-centred enterprise owned, controlled, and run by and for members to realise common economic, social, and cultural needs and aspirations.
+- **Clip.** The percentage Hypha skims from costs billed to client, to cover its operating costs. Example: "We operate with a 25% clip on our billed rate."
```

Comment:
```suggestion
- **Clip**: the percentage Hypha takes out of costs invoiced to clients in order to cover operating costs (e.g., "we clip 25% of our billable project work to fund administrative costs and collective programs"). See: [Calculating Member Wages](https://handbook.hypha.coop/finance.html#calculating-member-wages)
```

Bringing in line with language on finance page

Reply:
ping @patcon

Reply:
Thanks! Sounds good, but noting that this changed example from a noun to verb. "the clip" is how root systems spoke of it, and i think it's how we do, no? "the clip" or "a 25% clip" etc

Reply:
That example combines the language from the first and last sentence of the linked page, which is the only place I've seen it "officially" used:
> Each Member's pay from billable project work will have a percent-based portion taken from it to fund administrative costs and collective programs. ... The amount is calculated based on the invoiced amount before transaction costs. This clip, currently set at 25%, is the same across all projects during a period and can be revised as necessary by consensus.

In meetings we've used it both as a noun and verb to my recollection, and I don't seen an issue using with both noun and verb forms. If you want to keep it only as a noun, I've modified two words to make it so

Reply:
There is a PR that changes the referenced part, which I just merged. So here is new proposal:

```suggestion
- **Clip** a percent-based portion deducted from billable project work to fund administrative costs and collective programs of the Co-operative. See: [Calculating Member Wages](https://handbook.hypha.coop/finance.html#calculating-member-wages)
```

Reply:
I've hidden my suggestion. I'm happy to go ahead with @benhylau 's updated suggestion